cms/static/sass/views/_certificates.scss

```diff
@@ -0,0 +1,726 @@
+// studio - views - certificates
+// ====================
+
+// Table of Contents 
+// * +Layout - Certificates
+// * +Main - Collection
+// * +Main - Certificate
+// * +Signatories -Certificate
+
+
+// * +Layout - Certificates
+// ====================
+.view-certificates {
+
+  .content-primary, .content-supplementary {
+    @include box-sizing(border-box);
+    float: left;
+  }
+
+  .content-primary {
+    width: flex-grid(9, 12);
+    margin-right: flex-gutter();
+
+    .notice-moduledisabled {
+      @extend %ui-well;
+      @extend %t-copy-base;
+      background-color: $white;
+      padding: ($baseline*1.5) $baseline;
+      text-align: center;
+    }
+
+    .no-content {
+      @extend %no-content;
+    }
+
+    .wrapper-certificate {
+
+      .title {
+        @extend %t-title4;
+        @extend %t-strong;
+        margin-bottom: ($baseline/2);
+      }
+
+      .copy {
+        @extend %t-copy-sub1;
+      }
+    }
+  }
+
+  .content-supplementary {
+    width: flex-grid(3, 12);
+  }
+}
+
+// * +Main - Collection
+// ====================
+.view-certificates {
+
+  .wrapper-collection {
+    @extend %ui-window;
+    position: relative;
+    outline: none;
+
+    &:hover .collection .actions {
+      opacity: 1.0;
+    }
+
+    .collection-details {
+      padding: $baseline ($baseline*1.5);
+    }
+
+    .collection-header {
+      margin-bottom: 0;
+      border-bottom: 0;
+      padding-bottom: 0;
+    }
+
+    .collection-info {
+      @extend %t-copy-sub1;
+
+      &.certificate-info-inline {
+        margin: ($baseline/2) 0;
+      }
+
+      &.collection-info-inline {
+        display: table;
+        width: 70%;
+        margin: ($baseline/4) 0 ($baseline/2) $baseline;
+
+        li {
+          @include box-sizing(border-box);
+          display: table-cell;
+          margin-right: 1%;
+          padding: ($baseline/4) 0;
+        }
+      }
+    }
+
+    .collection-items {
+      margin-left: $baseline;
+      margin-bottom: ($baseline*0.75);
+
+      .item {
+        @extend %t-copy-lead1;
+        padding: ($baseline/7) 0 ($baseline/4);
+        border-top: 1px solid $gray-l4;
+        white-space: nowrap;
+
+        &:first-child {
+          border-top: none;
+        }
+
+        .name {
+          overflow: hidden;
+          text-overflow: ellipsis;
+          display: inline-block;
+          vertical-align: middle;
+          width: 75%;
+          margin-right: 5%;
+        }
+      }
+    }
+
+    .collection-details {
+
+      .actions {
+        @include transition(opacity .15s .25s ease-in-out);
```

Comment:
As @clrux  noted in a separate PR, this is out of order. Should be @include transition(opacity .15s ease-in-out .25s);